package.json

```diff
@@ -46,6 +46,7 @@
   "dependencies": {
     "history": "^3.0.0",
     "lodash.assign": "^4.2.0",
+    "lodash.isequal": "^4.4.0",
```

Comment:
On this note (but tangential to this PR) you may want to use `Object.assign` instead of `lodash.assign` if the dep tree is a concern here.